.github/workflows/e2e_tests.yml

```diff
@@ -140,12 +140,17 @@ jobs:
             ${{ runner.os }}-build-${{ env.cache-name }}-
             ${{ runner.os }}-build-
             ${{ runner.os }}-
+      - id: check-risk
```

Comment:
Qq -- we have a check-risk step above. Should we just be reusing that info here?

Reply:
Where exactly? If you mean line 45 then that is a different job and we cannot access it from here (or we can by using artifacts but that is more complicated than just running it again).

Reply:
Ah, I see. Yup it's line 45, but here's what I'm confused about -- if you look at https://github.com/oppia/oppia/pull/16758/files#diff-37aed4ad4a8c3a4220c5e3c2eee33093073b8d64f480cc742b63764c859b6c72R64, that step doesn't have an "if not low-risk" conditional. So why isn't the artifact there? 

I see the step above it does have the conditional so maybe that suffices to prevent the files being uploaded but should we add the check there too so it's easier for devs to understand what's going on?